styles/octopus.css

```
@@ -0,0 +1,72 @@
+.octopus {
+    height: 100vw;
+
+    padding-top: 90px;
+
+    background: 
+    linear-gradient(252.43deg, #D45B78 0.01%, #4B3862 101.01%);
+}
+
+.octopus__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.octopus__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 1280px;
+
+    margin-bottom: 14px;
+}
+
+.octopus__title {
+    color: #FCFCFC;
+
+    margin-bottom: 37px;
+}
+
+.octopus__subtitle {
+    width: 890px;
+    height: 72px;
+
+    text-align: center;
+    color: #F2F2F2;
+
+    margin-bottom: 68px;
+}
+
+.octopus__button {
```

```suggestion
.button-contained {
```
It is the independent BLOCK